src/onboarding/Deployment/DeploymentStepsPanel.js

```diff
@@ -1,11 +1,14 @@
 import React from 'react'
 import PropTypes from 'prop-types'
-import { textStyle, GU, useTheme, ProgressBar, Info } from '@aragon/ui'
+import { textStyle, GU, Link, useTheme, ProgressBar, Info } from '@aragon/ui'
 import DeploymentStepsItem from './DeploymentStepsItem'
 import { TransactionStatusType } from '../../prop-types'
+import { web3Providers } from '../../environment'
+import { identifyProvider } from '../../ethereum-providers'
 
 function DeploymentStepsPanel({ transactionsStatus, pending, allSuccess }) {
   const theme = useTheme()
+  const provider = identifyProvider(web3Providers.wallet)
```

Comment:
You can also get this information from `useWallet()`, which is recommended since we will likely be moving to dynamic user wallets in the future (to support multiple wallet connections).